docs/core/howto/ssl.rst

```diff
@@ -204,10 +199,40 @@ For example,
     options = CertificateOptions(..., dhParameters=dhParams)
 
 Another part of the TLS protocol which ``CertificateOptions`` can control is the version of the TLS or SSL protocol used.
-This is often called the context's "method".
+There are two ways to configure it, either by configuring a list of protocols, or by the legacy way of setting the internal context's "method".
+By default, Twisted will configure it to use TLSv1.0 or later and disable the insecure SSLv3 protocol, but this manual control over protocols can be helpful if you need to support legacy SSLv3 systems, or you wish to restrict it down to just the strongest of the TLS versions.
+
+You can give ``CertificateOptions`` a range of TLS versions you wish it to negotiate with the ``tlsProtocols`` argument in the initializer:
+
+.. code-block:: python
+
+    from twisted.internet.ssl import (CertificateOptions, TLSVersion,
+                                      TLSVersionRange)
+    options = CertificateOptions(
+        ...,
+        tlsProtocols=TLSVersionRange(TLSVersion.TLSv1_0,
+                                     TLSVersion.TLSv1_2))
+
```

Comment:
> there is essentially zero utility in being able to exclude a TLS version in the middle of your supported range.

I think the point of this feature is to allow users to express whatever version choice they want.  Saying "TLSv1.0 to newest TLS, but not TLSv1.1" might be an extreme edge case but the point of the new API is to support all the edge cases.  The _current_ API already does a reasonable job letting you deal with the common case of "newest support TLS version".  This change is about providing a mechanism for doing better than that in the edge cases.